standards/X509-SVID.md

```diff
@@ -56,7 +56,7 @@ Leaf and signing certificates carry different X.509 properties - some for securi
 ### 4.1. Basic Constraints
 The basic constraints X.509 extension identifies whether the certificate is a signing certificate, as well as the maximum depth of valid certification paths that include this certificate. It is defined in [RFC 5280, section 4.2.1.9][3].
 
-Valid X.509 SVIDs (both leaf and signing certificates) MUST NOT set the `pathLenConstraint` field. Signing certificates MUST set the `CA` field to `true`, and leaf certificates MUST set the `CA` field to `false`.
+SVID signing certificates MUST set the `CA` field to `true` and MUST NOT set the `pathLenConstraint` field. Leaf certificates MUST either omit the basic constraints extension or have an empty basic constraints field (an empty basic constraints field is the only correct encoding of a basic constraint with `CA` set to false.)
```

Comment:
@rra Right, the DER rules make it an error to explicitly encode CA = false. See https://github.com/awslabs/certlint/issues/11 for more context about why I suggested rewording things.